metricbeat/module/elasticsearch/index/data_test.go

```diff
@@ -50,7 +51,9 @@ func TestMapper(t *testing.T) {
 
 	httpClient, err := helper.NewHTTPFromConfig(helper.Config{
 		ConnectTimeout: 30 * time.Second,
-		Timeout:        30 * time.Second,
+		Transport: httpcommon.HTTPTransportSettings{
```

Comment:
why not use default here? 
tests in a lot of cases can serve as code examples

Reply:
I did just update an existing test in the ES output, and didn't want to give it the chance to maybe change some assumed behavior... not sure... was to lazy to read the test code.

As follow up we should rather add tests to the `httpcommon` package, so the code examples are where one would expect them and not scattered around in libbeat.